README.md

```diff
@@ -7,7 +7,8 @@
 
 ## Summary
 
-Config helps you easily manage environment specific settings in an easy and usable manner.
+Config helps you easily manage environment specific settings in an easy and
```

Comment:
Could you please revert all those new line changes from your editor? We do not use hard wrapping...

Reply:
@pkuczynski Done.  I was just following the linter errors.  May want to disable that one if you don't expect it to be fixed?